HTML/nested.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nested Classes Example</title>
+</head>
+<body>
+    <h1>Nested Classes Example</h1>
+    <div id="output"></div>
+
+    <script>
+        class OuterClass {
+            constructor(name) {
+                this.name = name;
+            }
+
+            greet() {
+                console.log(`Hello from ${this.name}`);
+            }
+
+            class InnerClass {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2188733227>#026393</a><span> - 06/25/2024, 11:52 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Incorrect syntax for defining a class within another class in JavaScript. JavaScript does not support nested class definitions in the manner shown.
</div>
<div id="fix">
<b>Fix:</b> Define the 'InnerClass' outside of the 'OuterClass' and use it appropriately.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -21,11 +21,12 @@
      class OuterClass {
          constructor(name) {
              this.name = name;
          }
 
          greet() {
              console.log('Hello from ${this.name}');
          }
      }
 
 -    class InnerClass {
 +    class InnerClass {
          constructor(innerName) {
              this.innerName = innerName;
          }
 
          innerGreet() {
              console.log('Hello from inner class, ${this.innerName}');
          }
      }
 
      const outer = new OuterClass('Outer');
      outer.greet();
 
 -    const inner = new OuterClass.InnerClass('Inner');
 +    const inner = new InnerClass('Inner');
      inner.innerGreet();
 
      document.getElementById('output').innerText = 'Check the console for greetings from ${outer.name} and ${inner.innerName}';
```
</div>
</details>
</div>
<br>


</div>